proto/gz/msgs/material_color.proto

```diff
@@ -31,6 +31,16 @@ import "gz/msgs/header.proto";
 
 message MaterialColor
 {
+  /// \brief Entities to apply material color.
+  enum Apply
```

Comment:
I'd prefer if we changed this to `EntityMatch` or `EntityChoice`. Same with the field name, `apply` -> `entity_match` or `entity_choice`.